### ISSUE TITLE
[Concept Entry] Sklearn kernel-ridge-regression

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#4859]
### Reason/inspiration (optional)

We would like a new entry on the `Kernel Ridge Regression` concept under Sklearn. The entry should go in a new file under `docs/content/sklearn/concepts/kernel-ridge-regression/kernel-ridge-regression.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept (if applicable)
- An `Example` section that provides an example demonstrating the concept in use (if applicable)
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use (if applicable)

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by dueprincipati** [comment]
I'd like to claim this issue. I have reviewed the task requirements and am prepared to work on this. I will follow the contribution guidelines and submit a pull request once completed. Please let me know if any additional information is needed from me. Thank you!

**Comment by mamtawardhani** [comment]
Hey @dueprincipati, do you want to take this issue as well? I've already assigned one issue to you.

**Comment by uhck** [comment]
Hi, I can take this issue. Thanks!

**Comment by SaviDahegaonkar** [comment]
Hey @uhck, Congratulations!  You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

Thanks  & Regards,
Savi

**Comment by uhck** [comment]
@SaviDahegaonkar @mamtawardhani Can you take a look at the open PR? It's been sitting for 2 weeks with no updates and I'd like to close this soon.

**Comment by mamtawardhani** [comment]
> @SaviDahegaonkar @mamtawardhani Can you take a look at the open PR? It's been sitting for 2 weeks with no updates and I'd like to close this soon.

@SaviDahegaonkar as I can see you are doing the first review, could you please check and update the labels if changes requested by you are made?

**Comment by SaviDahegaonkar** [comment]
@mamtawardhani Yes I am already checking on it, previously it was kn a invalid label but now the author has made the changes to make the file valid, I am already reviewing it and will get it done.
